app/components/containers/containersController.js

```diff
@@ -239,17 +241,23 @@ angular.module('containers', [])
         return;
       }
     });
+    var title = 'You are about to remove one or more container.';
     if (isOneContainerRunning) {
-      ModalService.confirmDeletion(
-        'You are about to remove one or more running containers.',
-        function (confirmed) {
-          if(!confirmed) { return; }
-          $scope.removeAction();
-        }
-      );
-    } else {
-      $scope.removeAction();
+      title = 'You are about to remove one or more running containers.';
     }
+    var text = 'Automatically clean associated volumes<i></i>';
```

Comment:
Please replace the text 'Automatically clean associated volumes' with 'Automatically remove non-persistent volumes'.